src/mainboard/starfive/visionfive1/main/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[package]
+name = "starfive-visionfive1-main"
+version = "0.1.0"
+edition = "2021"
+authors = [
+    "Daniel Maslowski <info@orangecms.org>",
+]
+
+[dependencies]
+embedded-hal = "1.0.0-alpha.8"
+nb = "1"
+spin = "0.9"
+# NOTE: until a newer release than 0.8.0 is tagged, use this to have the fix for
+# reading marchid/mimpid
+riscv = { git = "https://github.com/rust-embedded/riscv", commit = "e38a68d" }
```

Comment:
We have 0.9.0 released and it includes thisfix :)

Reply:
updated 🥳